exposee-ui/src/components/Registration/Registration.jsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState, useContext } from 'react';
+import { Link, useNavigate } from 'react-router-dom';
+import '../Registration/Registration.css'
+import { UserContext } from '../../UserContext';
+
+const Registration = () => {
+  const [username, setUsername] = useState('');
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+
+
+  // const updateUser  = 
+  // useContext(UserContext);
+  //const navigate = useNavigate();
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+
+    try {
+      // Make the signup API request
+      const response = await fetch(`http://localhost:3000/user`, {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({ username, email, password}),
```

Comment:
how is the username, email and password being passed into the POST request?

Reply:
it is passed as a body so like the json.stringify() function is used to convert the user object into JSON string representation.